tipg/settings.py

```diff
@@ -86,6 +86,7 @@ class FeaturesSettings(BaseSettings):
 
     default_features_limit: int = Field(10, ge=0)
     max_features_per_query: int = Field(10000, ge=0)
+    sort_columns: bool = True
```

Comment:
I think we should store this settings in the `TableSettings` class

Reply:
Agreed.